db/postgres.go

```diff
@@ -30,6 +30,12 @@ const (
 //go:embed postgres
 var sql embed.FS
 
+type ExtraIndexes struct {
+	Type  string
```

Comment:
Se os valores aqui não `"root"` e `"no_root"` essa estrutura de dados deveria ser `boolean`, não `string`.

```suggestion
	IsRoot  boolean
```